androidapp/app/build.gradle

```diff
@@ -45,8 +45,13 @@ dependencies {
     implementation 'com.google.android.material:material:1.1.0'
     implementation 'androidx.work:work-runtime-ktx:2.3.4'
     implementation 'com.google.code.gson:gson:2.8.5'
+    implementation 'com.squareup.moshi:moshi:1.9.2'
+    implementation 'com.squareup.moshi:moshi-kotlin:1.9.2'
+
+    annotationProcessor 'com.squareup.moshi:moshi-kotlin-codegen:1.9.2'
 
     testImplementation 'junit:junit:4.12'
+    debugImplementation 'com.amitshekhar.android:debug-db:1.0.6'
```

Comment:
Is this the library that I mentioned for visualizing seeing a Room database?

Reply:
Yeah, haven't gotten it to work yet

Reply:
I wonder if it has something to do with the firewall settings on your laptop. I'll need to try it on my laptop.